### PR TITLE
Improve introduction about disassembling and document `asm.cpu`

### PR DESCRIPTION
**Pull request by officialcjunior**
* Added a short explanation about `asm.cpu`.
* Improve documentation about introduction on disassembling: added documented `pdp` and `pdj`command.
* Fixed the link at the chapter Getting Started. The link originally pointed to `rizin.org`.

### CODE REVIEW

File: src/first_steps/getting_rizin.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Downloading rizin
 
-You can get rizin from the website, [http://rizin.org](http://rizin.org),
+You can get rizin from the website, [http://rizin.re](http://rizin.re),
```
Comment (ITAYC0HEN) [inline src/first_steps/getting_rizin.md:3]: That's currently untrue

**Review by ITAYC0HEN — CHANGES_REQUESTED** [review]
nice!

File: src/first_steps/getting_rizin.md (hunk 1 above)
Comment (officialcjunior) [reply src/first_steps/getting_rizin.md:3]: So, I'll get it back to `rizin.org`. That's what you meant, right?

**Review by ITAYC0HEN — APPROVED** [review]
Thank you!

**Pull request merged**